assets/Js/script.js

```diff
@@ -0,0 +1,68 @@
+const sidebar = document.querySelector('.left-sidebar');
+const hidehamburgerL = document.querySelector('#hamburger-L');
+const hidehamburgerR = document.querySelector('#hamburger-R');
+const hidecrossL = document.querySelector('#cross-icon-L');
+const hidecrossR = document.querySelector('#cross-icon-R');
+const navbar = document.querySelector('.navbar');
+const Darkmode = document.querySelector('.dark-mode');
```

Comment:
here as well rename it as darkMode

Reply:
it is in red color bcz its not in camel case?